config/config.go

```diff
@@ -62,3 +66,22 @@ func LoadClusterConfig() (utilsmetadata.ClusterConfig, error) {
 
 	return *clusterConfig, err
 }
+
+func UpdateClusterConfigURLs(clusterConfig *utilsmetadata.ClusterConfig, filePath string) error {
+	pathAndFileName, present := os.LookupEnv("SERVICES")
+	if !present {
+		pathAndFileName = filePath
+	}
+	logger.L().Debug("discovery service URLs from file", helpers.String("path", pathAndFileName))
+
+	services, err := servicediscovery.GetServices(
+		v1.NewServiceDiscoveryFileV1(pathAndFileName),
+	)
+	if err != nil {
+		return err
+	}
+
+	clusterConfig.EventReceiverRestURL = services.GetReportReceiverHttpUrl()
```

Comment:
let's remove these URLs from clusterConfig and use directly the service discovery file